src/lang/english.txt

```diff
@@ -3057,7 +3057,7 @@ STR_NEWGRF_BROKEN_CAPACITY                                      :{WHITE}It chang
 STR_BROKEN_VEHICLE_LENGTH                                       :{WHITE}Train '{VEHICLE}' belonging to '{COMPANY}' has invalid length. It is probably caused by problems with NewGRFs. Game may desync or crash
 
 STR_NEWGRF_BUGGY                                                :{WHITE}NewGRF '{0:RAW_STRING}' provides incorrect information
-STR_NEWGRF_BUGGY_ARTICULATED_CARGO                              :{WHITE}Cargo/refit information for '{1:ENGINE}' differs from purchase list after construction. This might cause autorenew/-replace to fail refitting correctly
+STR_NEWGRF_BUGGY_ARTICULATED_CARGO                              :{WHITE}Cargo/Refit information for '{1:ENGINE}' differs from purchase list after construction. This might cause autorenew/-replace to fail refitting correctly
```

Comment:
Wrong, IMO.

Reply:
Should this be `autorenew/autoreplace` (without the dash), or should the dash stay (`autorenew/-replace`)?